tests/chainer_tests/links_tests/normalization_tests/test_layer_normalization.py

```diff
@@ -21,86 +20,6 @@ def _create_ln(*args, **kwargs):
         chainer.disable_experimental_feature_warning = flag
 
 
-@testing.parameterize(*(testing.product({
-    'batchsize': [1, 5],
-    'size': [10, 20],
-    'dtype': [numpy.float32],
-})))
-class LayerNormalizationTest(unittest.TestCase):
```

Comment:
It seems to me that we don't have to remove this test case. Could you tell me the reason?

Reply:
I removed this test because exactly the same thing is tested in `functions_tests`.

Reply:
It is true that the functionality of `L.LayerNormalization` and `F.layer_normalization` overlaps much, but I think this test checks what the test added in this PR does not check (e.g. registration of parameters `beta` and `gamma` to the link).

Reply:
Hmm, which test?

Reply:
I meant, `TestLayerNormalization`.

Reply:
I mean, which specific test method which are removed from `links_tests` and not tested in `functions_tests`?
What I removed is tests for forward, backward, and double-backward calculation.
These are all tested in `functions_tests`.